gui_v1.py

```diff
@@ -265,35 +309,35 @@ def launcher(self):
                                     "pm",
                                     "f0method",
                                     key="pm",
-                                    default=data.get("pm", "") == True,
+                                    default=data.get("pm", False),
                                     enable_events=True,
                                 ),
                                 sg.Radio(
                                     "harvest",
                                     "f0method",
                                     key="harvest",
-                                    default=data.get("harvest", "") == True,
+                                    default=data.get("harvest", False),
                                     enable_events=True,
                                 ),
                                 sg.Radio(
                                     "crepe",
                                     "f0method",
                                     key="crepe",
-                                    default=data.get("crepe", "") == True,
+                                    default=data.get("crepe", False),
                                     enable_events=True,
                                 ),
                                 sg.Radio(
                                     "rmvpe",
                                     "f0method",
                                     key="rmvpe",
-                                    default=data.get("rmvpe", "") == True,
+                                    default=data.get("rmvpe", False),
                                     enable_events=True,
                                 ),
                                 sg.Radio(
                                     "fcpe",
                                     "f0method",
                                     key="fcpe",
-                                    default=data.get("fcpe", "") == True,
+                                    default=data.get("fcpe", True),

```

Comment:
I think this value is false.

Reply:
Sure.

Reply:
@yxlllc Please check here

Reply:
Fcpe has a very strong performance and is much faster than rmvpe so it can be the default

Reply:
Well, then no problem since that's your purpose.